packages/rs-dpp/src/voting/contender_structs/mod.rs

```diff
@@ -133,6 +133,11 @@ impl TryFrom<ContenderWithSerializedDocument> for FinalizedContenderWithSerializ
 /// This struct holds the identity ID of the contender, the serialized document,
 /// and the vote tally.
 #[derive(Debug, PartialEq, Clone, Default)]
+#[cfg_attr(
+    feature = "document-serde-conversion",
```

Comment:
this shouldn't be document-serde-conversion

Reply:
without `document-serde-conversion` this fails due to lack of serde on Document.

Should I create new feature?

Reply:
Removed this code.